azext_iot/operations/dps.py

```diff
@@ -223,6 +238,8 @@ def iot_dps_device_enrollment_group_update(client,
                                            etag,
```

Comment:
Consider removing the requirement for etag (like in the enrollment group update command). You don't need to follow the generic update process to achieve. In the current versions of the iot extension, the iot hub based commands don't require etag as it is derived wherever necessary. This is one less parameter users have to deal with (especially when some commands have tons of overloads).

Reply:
Do we consider the possibility that multiple users running the command at the same time?

Reply:
@YingXue and I discussed this topic and determined the best compromise was to have etag be an optional parameter.